From d1cafd2b0059511856cab1d884ef501768ac104d Mon Sep 17 00:00:00 2001
From: maniacx <macs18max@gmail.com>
Date: Sun, 2 Aug 2015 00:06:51 +0400
Subject: [PATCH 2/3]  disable block-based ota

Disable it for those whose network bandwidth is less than their device's emmc bandwidth.

Change-Id: Icdc4af33f83bdbffef60c6492c9887ce42f92e8c
---
 core/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/core/Makefile b/core/Makefile
index a8a8b6b..5fd2c81 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -1708,7 +1708,6 @@ $(INTERNAL_OTA_PACKAGE_TARGET): $(BUILT_TARGET_FILES_PACKAGE) $(DISTTOOLS)
 	@echo -e ${CL_YLW}"Package OTA:"${CL_RST}" $@"
 	$(hide) MKBOOTIMG=$(MKBOOTIMG) \
 	   $(OTA_FROM_TARGET_SCRIPT) -v \
-	   --block \
 	   $(if $(WITH_LZMA_OTA), -z) \
 	   -p $(HOST_OUT) \
 	   -k $(KEY_CERT_PAIR) \
-- 
2.1.4

